### ISSUE TITLE
Shift Logout button to the bottom

### ISSUE THREAD

**Issue by ishitar** [anuraghazra/convoychat#38]
Whenever more rooms are created, logout button keeps popping down. For a cleaner UI, I suggest keeping the Logout button at the bottom.
We can create another issue for making the view of rooms scrollable if they exceed a certain number. 
![Screenshot 2020-06-30 at 12 24 31 AM](https://user-images.githubusercontent.com/41416752/86044998-922cdc00-ba68-11ea-9112-7ef6439b6117.png)

**Comment by anuraghazra** [comment]
Hi @ishitar actually that login button is temporary for now. I'll move that to somewhere else later. And also probably change it to an icon from button.

**Comment by ishitar** [comment]
Yeah that'll be better, where do you plan on moving it? I could do that and change it to an icon as well.

**Comment by anuraghazra** [comment]
@ishitar i'm really not sure... i have to find an optimal place for it.. i'll go ahead and tinker with the figma design to see where it will work nicely.
